pkg/machine/compression/decompress.go

```diff
@@ -19,6 +19,7 @@ const (
 	macOs                = "darwin"
 	progressBarPrefix    = "Extracting compressed file"
 	zipExt               = ".zip"
+	magicNumberMaxSize   = 10
```

Comment:
This seems fine for now but in the long term I guess we should move this definition to the place where we actually check the bytes, c/storage/pkg/archive.